src/transition/transition.ts

```diff
@@ -265,7 +266,8 @@ export class Transition implements IHookRegistry {
    */
   views(pathname: string = "entering", state?: State): ViewConfig[] {
     let path = this._treeChanges[pathname];
-    return state ? find(path, propEq('state', state)).views : unnest(path.map(prop("views")));
```

Comment:
These views are now ViewConfig objects, not the raw ViewDeclarations from the state.views block